### DISCUSSION TITLE
Town Hall Meetings - Alternate Times

### DISCUSSION THREAD

**Discussion by phillewis** [Council-of-DAOs/Council-Hub#8] [Project Catalyst]
I raised this on IdeaScale in the Project Catalyst Problem Sensing challenge at https://cardano.ideascale.com/a/dtd/Weekly-townhall-times-not-varied/323792-48088.  I have also raised it as a question in the town hall itself and on the Catalyst Telegram group.

Essentially, while ever there is a live element to the town hall (I believe they are trialing a new interactive process that is separate to the town hall itself), so as not to exclude people in the Asia Pacific region (UTC+6 to UTC+10), the town hall start time should be alternated each week.  The required change could still be around the IOHK staff availability (i.e. within their day).

**Comment by stephen-rowan** [comment]
Hi @phillewis - I have also referred to this problem as an improvement example in this Catalyst Discord post (https://discordapp.com/channels/756943420660121600/850274937666404382/852319204049027103).

It is also relevant to the accessibility of the "Open Space" Swarm event launched at TH on 09/06/21 (https://github.com/Council-of-DAOs/Council-Hub/discussions/14) 

Hopefully we can keep raising this issue to encourage Project Catalyst to extend its global reach.
